js/views/pageNavVw.js

```diff
@@ -91,17 +96,48 @@ module.exports = Backbone.View.extend({
   },
 
   handleSocketMessage: function(response) {
-    var data = JSON.parse(response.data);
-    if(data.id == this.socketNotificationID){
-      console.log(data);
-    }
-  },
+    var data = JSON.parse(response.data),
+        username,
+        avatar,
+        n;
+
+    if (data.hasOwnProperty('notification')) {
+      n = data.notification;
+      username = n.handle ? n.handle : n.guid.substring(0,10) + '...';
+      avatar = n.image_hash ? app.serverConfig.getServerBaseUrl + '/get_image?hash=' +
+        n.image_hash + '&guid=' + n.guid : 'imgs/defaultUser.png';
+
```

Comment:
hmm... yeah, a lot of this code I just re-used and didn't tweak too much. I'll think about updating the model. I'm quite hesitant because in general we're not using models correctly. Any state that is purely for display purposes should really either be switched by the view or done in the template. Models are pieces of data that are intended to be used by many different views and we shouldn't be corrupting them with display state that is only relevant for some views. For example, we may want to pass the model to a view that spans the width of the browser which would want the full username.

It's a very common practice in a backbone app to have models and collections provided by the router (or some centralized place). That way the data can be passed to any number of views. The centralized place handles the fetching so each view that requires the same data doesn't need to instantiate new models or collections and hit the server for data that we already have.